models/Mongoschema.js

```diff
@@ -8,37 +8,51 @@ const questionnaireSchema = new Schema(
       quality: {
         type: String,
         enum: ['terrible', 'bad', 'neutral', 'satisfied', 'excelent'],
-        required: true,
+        required: [true,'sleep quality must be indicated']
       },
       quantity: {
-        goToBed: { type: String, required: true },
-        wakeUp: { type: String, required: true },
-        total: { type: Number, required: true },
+        goToBed: { type: String, required: [true,'bedtime must be indicated'] ,
+        validate: {
+          validator: function(v) {
+              var time = /^([0-1]?[0-9]|2[0-3]):[0-5][0-9]$/;
+              return (!v || !v.trim().length) || time.test(v)
```

Comment:
What happen when the value `v` is `undefined` ? and when `v is ""`? Are them valid values for `goToBed`?